### ISSUE TITLE
How to finetune the last 2 fc layers？

### ISSUE THREAD

**Issue by sophiazy** [davidsandberg/facenet#930]

**Comment by shoyo-su** [comment]
Hey Sophiazy to finetune the last 2 layers I cannot give an accurate solutions without seeing your whole model,but you could try some things...
1)make the activation function of the second last layer tanh it has an advantage over the sigmoidal activation fuction in such conditions .
2)then add a drop out layer to avoid overfitting.
3)then in the in the last layer it will by default be a softmax activation function.
4)try keeping the no of neurons in second last layer double of that in the last layer that is the no of classes you want to categorise into.
